std/experimental/checkedint.d

```diff
@@ -1366,8 +1366,13 @@ default behavior.
 */
 struct Warn
 {
-    import std.stdio : stderr;
+    import std.stdio : File;
 static:
+    private @property File trustedStderr() @trusted
```

Comment:
`File` is unsafe for a reason. `Checked!Warn` and `Checked!Abort` are currently thread unsafe and this will mark them as good to go.

I'm of the opinion that `trustedStdout` was a huge mistake we're still paying for.

Reply:
This is a `private` property and _only_ used for printing to stdout within `Warn`.